<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>871</width>
      <height>631</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>320</width>
      <height>320</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-image: url(:/resurses/images/fon.png);</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>кодировать </string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>выберите файл </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>70</y>
        <width>380</width>
        <height>400</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>380</width>
        <height>400</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>380</width>
        <height>380</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/resurses/images/fon2.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>380</width>
        <height>400</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>380</width>
        <height>400</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/resurses/images/fon2.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>31</x>
        <y>501</y>
        <width>621</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="encodeButton">
      <property name="geometry">
       <rect>
        <x>657</x>
        <y>501</y>
        <width>101</width>
        <height>29</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Кодировать </string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveButton">
      <property name="geometry">
       <rect>
        <x>759</x>
        <y>501</y>
        <width>81</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Скачать </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>декодировать </string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>450</width>
        <height>450</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>450</width>
        <height>450</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/resurses/images/fon2.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>511</x>
        <y>249</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>выберите файл</string>
      </property>
     </widget>
     <widget class="QPushButton" name="decodeButton">
      <property name="geometry">
       <rect>
        <x>511</x>
        <y>322</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Декодировать </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>511</x>
        <y>287</y>
        <width>311</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
